src/app.js

```diff
@@ -1,29 +1,26 @@
-//Importamos configuracion de Express
-import app from './config/express.js';
-
-//Importamos rutas
-import routes from './routes/index.route.js';
-
-import mongoose from 'mongoose';
-
+//Para usar require, hay que importar esto
 import { createRequire } from 'module';
-
 const require = createRequire(import.meta.url);
-
 require('dotenv').config();
```

Comment:
I recommend you use either `import` or `require`, but not both at the same time. You can use `dotenv` with import syntax.

```js
import { config } from 'dotenv'
config();
```
This is something you can configure when setting your project up with `eslint`